src/primitives/line/mod.rs

```diff
@@ -339,6 +339,7 @@ impl Transform for Line {
 }
 
 #[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
+#[allow(dead_code)] // TODO: remove when all variants are used
```

Comment:
All going well, this will be removed in the thick triangle work I'm doing at the moment. The warnings are annoying, but could you remove this TODO as I have a feeling we'll forget it in the next review.

Reply:
> All going well, this will be removed in the thick triangle work I'm doing at the moment.

Nice, I hope this will be less aggravating than the polylines.

> The warnings are annoying, but could you remove this TODO as I have a feeling we'll forget it in the next review.

OK.

Reply:
It's going well so far but as with all of these cursed features I'll probably spend 6 months on the last 0.01%!